HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -163,6 +163,14 @@ def customiseFor18429(process):
 
      return process
 
+# Add new parameters to RecoTrackRefSelector
+def customiseFor19029(process):
+    for producer in producers_by_type(process, "RecoTrackRefSelector"):
+        if not hasattr(producer, "minPhi"):
+            producer.minPhi = cms.double(-3.2)
+            producer.maxPhi = cms.double(3.2)
```

Comment:
This (adding two top-level parameters to a plugin) is better done by a fillDescriptions().